com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -107,6 +107,26 @@ internal class DeferredLights
     {
         public static class ShaderConstants
```

Comment:
not related to this PR but I just realized this could be made internal unless we have a good reason to expose these constants to users.

Reply:
Otherwise please add API docs.

Reply:
Made internal.